shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -824,11 +829,23 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
       result.setText(semanticsNode.getValueLabelHint());
     } else if (!semanticsNode.hasFlag(Flag.SCOPES_ROUTE)) {
       CharSequence content = semanticsNode.getValueLabelHint();
+      if (Build.VERSION.SDK_INT < Build.VERSION_CODES.P) {
+        if (semanticsNode.tooltip != null) {
+          content = content != null ? content : "";
+          content = content + "\n" + semanticsNode.tooltip;
```

Comment:
comment about why a new line and the intended behavior would be appreciated. 

Is it possible to test this logic?